### PR TITLE
Update four patch versions

### PR DESCRIPTION
**Pull request by lucaswerkmeister**
```bash
npm i {@types/lodash,@vue/test-utils,husky,lint-staged}@latest
```

Bug: T323899

### CODE REVIEW

**Comment by lucaswerkmeister** [comment]
The last dregs from `npm outdated`. There’s nothing that ties these four together, other than the fact that the version component being updated is the patch version.

File: package.json [hunk 1]
```diff
@@ -54,12 +54,12 @@
...
 		"@rushstack/eslint-patch": "^1.2.0",
 		"@types/jest": "^29.2.3",
-		"@types/lodash": "^4.14.187",
+		"@types/lodash": "^4.14.191",
 		"@typescript-eslint/eslint-plugin": "^5.45.0",
 		"@typescript-eslint/parser": "^5.45.0",
 		"@vitejs/plugin-vue": "^2.3.4",
 		"@vue/eslint-config-typescript": "^11.0.2",
-		"@vue/test-utils": "v2.2.1",
+		"@vue/test-utils": "^2.2.5",
```
Comment (lucaswerkmeister) [inline package.json:62]: Oh neat, a mirrored caret operator!

Comment (micgro42) [reply package.json:62]: it's fascinating that this still worked. I guess it was then effectively pinned to that particular version?

**Review by micgro42 — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (lucaswerkmeister) [reply package.json:62]: Looks like it, yeah – on https://semver.npmjs.com/ a leading “v” seems to be accepted and ignored.

**Pull request merged**